### ISSUE TITLE
Unable to determine repo name

### ISSUE THREAD

**Issue by esc** [AlexeySanko/pybuilder_semver_git_tag#5]
When there is no remote repository `origin` I get:

```
Traceback (most recent call last):
  File "/home/esc/git/check-pybuilder-semver-plugin/.venv3/lib/python3.5/site-packages/pybuilder/cli.py", line 405, in main
    exclude_all_optional=options.exclude_all_optional
  File "/home/esc/git/check-pybuilder-semver-plugin/.venv3/lib/python3.5/site-packages/pybuilder/reactor.py", line 108, in prepare_build
    self.project_module = self.load_project_module(project_descriptor)
  File "/home/esc/git/check-pybuilder-semver-plugin/.venv3/lib/python3.5/site-packages/pybuilder/reactor.py", line 334, in load_project_module
    return imp.load_source("build", project_descriptor)
  File "/usr/lib/python3.5/imp.py", line 172, in load_source
    module = _load(spec)
  File "<frozen importlib._bootstrap>", line 693, in _load
  File "<frozen importlib._bootstrap>", line 673, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 665, in exec_module
  File "<frozen importlib._bootstrap>", line 222, in _call_with_frames_removed
  File "/home/esc/git/check-pybuilder-semver-plugin/build.py", line 9, in <module>
    use_plugin('pypi:pybuilder_semver_git_tag')
  File "/home/esc/git/check-pybuilder-semver-plugin/.venv3/lib/python3.5/site-packages/pybuilder/core.py", line 215, in use_plugin
    reactor.require_plugin(name, version, plugin_module_name)
  File "/home/esc/git/check-pybuilder-semver-plugin/.venv3/lib/python3.5/site-packages/pybuilder/reactor.py", line 74, in require_plugin
    self.import_plugin(plugin, version, plugin_module_name)
  File "/home/esc/git/check-pybuilder-semver-plugin/.venv3/lib/python3.5/site-packages/pybuilder/reactor.py", line 199, in import_plugin
    plugin_module = self.plugin_loader.load_plugin(self.project, plugin, version, plugin_module_name)
  File "/home/esc/git/check-pybuilder-semver-plugin/.venv3/lib/python3.5/site-packages/pybuilder/pluginloader.py", line 144, in load_plugin
    return loader.load_plugin(project, name, version, plugin_module_name)
  File "/home/esc/git/check-pybuilder-semver-plugin/.venv3/lib/python3.5/site-packages/pybuilder/pluginloader.py", line 102, in load_plugin
    return self._load_installed_plugin(thirdparty_plugin, name)
  File "/home/esc/git/check-pybuilder-semver-plugin/.venv3/lib/python3.5/site-packages/pybuilder/pluginloader.py", line 123, in _load_installed_plugin
    plugin_module = _load_plugin(thirdparty_plugin, name)
  File "/home/esc/git/check-pybuilder-semver-plugin/.venv3/lib/python3.5/site-packages/pybuilder/pluginloader.py", line 194, in _load_plugin
    __import__(plugin_module_name)
  File "/home/esc/git/check-pybuilder-semver-plugin/.venv3/lib/python3.5/site-packages/pybuilder_semver_git_tag/__init__.py", line 242, in <module>
    Reactor.current_instance().project, Reactor.current_instance().logger)
  File "/home/esc/git/check-pybuilder-semver-plugin/.venv3/lib/python3.5/site-packages/pybuilder_semver_git_tag/__init__.py", line 230, in force_semver_git_tag_plugin
    project.name = _get_repo_name(_get_repo_path(project))
  File "/home/esc/git/check-pybuilder-semver-plugin/.venv3/lib/python3.5/site-packages/pybuilder_semver_git_tag/__init__.py", line 107, in _get_repo_name
    _get_repo(repo_path).remotes.origin.url).path)
  File "/home/esc/git/check-pybuilder-semver-plugin/.venv3/lib/python3.5/site-packages/git/util.py", line 889, in __getattr__
    return list.__getattribute__(self, attr)
AttributeError: 'IterableList' object has no attribute 'origin'
------------------------------------------------------------
BUILD FAILED - 'IterableList' object has no attribute 'origin'
------------------------------------------------------------
```

**Comment by AlexeySanko** [comment]
Hi, @esc 
I've added it to list of work.
Thanks a lot.

**Comment by AlexeySanko** [comment]
@esc , thanks a lot for contribution.
Fixed into [1.2.1](https://pypi.python.org/pypi/pybuilder-semver-git-tag/1.2.1)
